src/main/kotlin/com/atlassian/performance/tools/jiraactions/page/SingleSelect.kt

```diff
@@ -22,9 +22,12 @@ internal class SingleSelect(
             condition = elementToBeClickable(input)
         )
         inputElement.click()
+        // this is a workaround for the fact that clear() can be processed asynchronously and can remove
+        // some of the keys sent to the browser. We have tested that under stress load, sending one backspace
+        // before the value fixes this issue.
+        // Whether we lose the backspace or not, it doesn't matter for the overall result.
```

Comment:
Awesome finding!! ❤️🏆

Reply:
![b-e-a utiful!](https://user-images.githubusercontent.com/1194704/71897030-a1843a00-3155-11ea-9ef7-0dfd239e6ebd.png)
We'll conquer flakiness!